### DISCUSSION TITLE
How to move public player to different domain

### DISCUSSION THREAD

**Discussion by krono-i2** [AzuraCast/AzuraCast#5880] [Q&A] [answered]
Hello!
I need to move the azuracast public player to a different domain.
I don't know which folder of the repository I've to get and modify in order to obtain a running player.
Thanks a lot!

Ivan

**Comment by Mozitux** [comment, marked as answer]
You can use an Iframe to include the whole page of the public player on the right domain, or use the widget player if you just want to place the player on a specific area.

**Reply by krono-i2** [reply]
Thank you for answer!
Can you show me a code example using the widget player?
Thanks so much!

Ivan

**Comment by krono-i2** [comment]
Precisely, I want to clone the public player, in order to add functionality (e.g., specific service workers for PWA).
Thank you.

Ivan
